app/assets/javascripts/discourse/app/controllers/user-activity.js

```diff
@@ -35,6 +35,18 @@ export default Controller.extend({
       : I18n.t("drafts.label");
   },
 
+  @discourseComputed("currentUser.pending_posts_count")
+  pendingLabel(count) {
+    return count > 0
```

Comment:
Translation keys support this operation natively. You can make the translation key have a `none:` and `other:` subkey and it'll use the appropriate one.

Reply:
That’s what I thought at first but then Gerhard explained me why: https://github.com/discourse/discourse/pull/14501#discussion_r722258300 (and we have a linter in place to avoid using `zero`)

Reply:
Oops sorry! Thanks for the clarification.